### ISSUE TITLE
Kiwi Commerce tab has escaped html in admin panel.

### ISSUE THREAD

**Issue by mgwebgroup** [kiwicommerce/magento2-cron-scheduler#14]
### Preconditions
Magento version: 2.3.3
DB: MariaDB 10.2
PHP: 7.2.22
Apache: 2.4.38

installed  via composer from Packagist v1.0.5

### Steps to reproduce
Log in to admin back end. Go to Stores->Configuration. 

Title tab for Kiwi Commerce is displayed as:
`<span class="kiwiCommerce-logo">KiwiCommerce Extensions</span>`

View of source html in browser (Chrome and Firefox) indicates rendering of the tab as follows:
```
<div class="admin__page-nav-title title _collapsible" data-role="title">
  <strong>&lt;span class=&quot;kiwiCommerce-logo&quot;&gt;KiwiCommerce Extensions&lt;/span&gt</strong>
</div>
```

### Expected result
Span tags should not be escaped.

**Comment by hostep** [comment]
Confirmed on Magento 2.3.4, using version 1.0.5, is kind of ugly:
<img width="551" alt="Screenshot 2020-02-21 at 15 47 22" src="https://user-images.githubusercontent.com/85479/75044322-b6c4e600-54c1-11ea-808b-0c9c3c35afd1.png">

Is probably already fixed by https://github.com/kiwicommerce/magento2-cron-scheduler/commit/45ddd9a2ef480940320ee0822541eaf29a8ffc0e

@kiwi-commerce: can you guys tag a new version where this fix is included?

Thanks!

**Comment by kiwi-commerce** [comment]
@mgwebgroup @hostep We have released a new version 1.0.6.
Thank you.

**Comment by hostep** [comment]
Thanks @kiwi-commerce!

For some reason packagist doesn't see version 1.0.6 yet: https://packagist.org/packages/kiwicommerce/module-cron-scheduler
Can you update it over there as well?

(You can add some automation in packagist with webhooks to make sure this happens automatically, that's probably not setup yet?)
